dev-requirements.txt

```diff
@@ -283,9 +283,8 @@ requests==2.20.0 \
     --hash=sha256:99dcfdaaeb17caf6e526f32b6a7b780461512ab3f1d992187801694cba42770c \
     --hash=sha256:a84b8c9ab6239b578f22d1c21d51b696dcfe004032bb80ea832398d6909d7279 \
     # via -r requirements.in, securedrop-sdk
-securedrop-sdk==0.0.13 \
-    --hash=sha256:3d521e7a63cd7df55f9c6508010f968692eccdb222c57ed9c2d0390fbbbd6f99 \
-    --hash=sha256:7763bb44755bdfc387ab6c002cbe49eeec2611feb04a8787c3c9f2aa48a1ee5f \
+securedrop-sdk==0.1.0 \
+    --hash=sha256:970fde25e6238e1808ac120951ee972549f4cd7952966dfe29f731bb308cc0d8 \
```

Comment:
matches what we have in pypi

Reply:
also matches the 256 shasum for https://github.com/freedomofpress/securedrop-debian-packaging/blob/master/localwheels/securedrop-sdk-0.1.0.tar.gz